src/ida_pro_mcp/idalib_mcp_server.py

```diff
@@ -0,0 +1,92 @@
+import sys
+import logging
+import argparse
+import importlib
+from pathlib import Path
+
+from rich.console import Console
+from rich.logging import RichHandler
+from mcp.server.fastmcp import FastMCP
+from mcp.server.fastmcp.server import Settings
+
+# idapro must go first to initialize idalib
+import idapro
+
+import ida_auto
+import ida_hexrays
+
+logger = logging.getLogger(__name__)
+
+
+def main() -> int:
+    parser = argparse.ArgumentParser(description="MCP server for IDA Pro via idalib")
+    parser.add_argument("--verbose", action="store_true", help="Show debug messages")
+    parser.add_argument("--transport", choices=["stdio", "sse"], default="stdio", help="MCP protocol to use")
```

Comment:
What do you think about unifying this into a single parameter? `--transport stdio` vs `--transport http://127.0.0.1:8000` Probably `--transport` is not the right name in that case though.

Reply:
thats reasonable.

an alternatively might be to graduate the transport and details into a required positional argument, perhaps like:

    python -m ida_pro_mcp.idalib_mcp_server /path/to/file.exe stdio
    python -m ida_pro_mcp.idalib_mcp_server /path/to/file.exe http://localhost:8000

Reply:
I thought about it a bit more and perhaps it will be cleaner to let the presence of `--address` or `--port` imply SSE and without them we pick stdio?

Reply:
I find that to be a little implicit, which can be fixed either with more help output, or perhaps renaming the arguments a little more.

How about an optional argument `--transport` with default value of `--transport=stdio`, but can be overridden to be `--transport=sse://localhost:8000`. This way, the default does what most people expect, but there's a way to concisely pick SSE with configuration.

In any case, I don't feel too strongly about any of this, and am happy to go along with whatever you prefer.

Reply:
Yeah doing `--transport=sse://localhost:8000` is good for me!